mitodl/open-discussions
# add support for material radio button

#### What are the relevant tickets?

part of #451 

#### What's this PR do?

This adds the material component for the radio button and imports the CSS for it, and also bumps us up to the latest version of our `mld-react-components` package.

#### How should this be manually tested?

It is just a dependency change, so there is no functionality to test. If you want to test that the package is added and the `<Radio />` component is showing up you can

- check out this branch
- revert commit `c445a734c1f8166629d9964c7e80d92581c2023b`
- visit the channel create page

You should see a material-style radio button there.
